src/infection-model/seir.tsx

```diff
@@ -56,6 +57,13 @@ export const seirIndexList = Object.keys(seirIndex)
   // these should all be numbers anyway but this extra cast makes typescript happy
   .map((k) => parseInt(seirIndex[k as any]));
 
+// though obscure, this has the effect of filtering out the reverse-mapping keys
```

Comment:
Crazy obscure! I would almost argue for replacing it with a simple function that does the filtering, unless you feel strongly about this.

Reply:
i'm not so sure that would be any better .... the alternative to this in typescript-land would be just manually creating a union type of all the enum keys that we would have to keep in sync with the enum by hand. A function that could serve as a type guard would probably still require that, plus we would have a function that we would have to call all over the place (which is not currently the case) rather than just relying on static analysis as here. 

This is super hacky though, so if you would be more comfortable with an explicit lists and loud comments about keeping it in sync, I can get with that

Reply:
No worries, we can stick with it as is.